pkg/reconciler/revision/config/store_test.go

```diff
@@ -81,16 +81,6 @@ func TestStoreLoadWithContext(t *testing.T) {
 		if diff := cmp.Diff(expected, config.Observability); diff != "" {
 			t.Error("Unexpected observability config (-want, +got):", diff)
 		}
-
```

Comment:
There's another test called `TestStoreImmutableConfig` failing after this change.